packages/salesforcedx-apex-replay-debugger/package.json

```diff
@@ -5,6 +5,7 @@
     "Implements the VS Code Debug Protocol for the Apex Replay Debugger",
   "version": "42.18.0",
   "publisher": "salesforce",
+  "preview": true,
```

Comment:
@JimSuplizio - Just FYI. We missed this when you made this extension publishable.